src/diffusers/models/embeddings.py

```diff
@@ -352,7 +352,7 @@ def token_drop(self, labels, force_drop_ids=None):
         labels = torch.where(drop_ids, self.num_classes, labels)
         return labels
 
-    def forward(self, labels, force_drop_ids=None):
+    def forward(self, labels: torch.LongTensor, force_drop_ids=None):
```

Comment:
hey @Birch-san PR's great :) just in the future it'd be super helpful to not add type annotations if the argument isn't a part of the PR. No issues with adding type annotations you feel are important in a separate PR. It just helps us to be able to quickly look at a diff of the function definition and see what arguments were added/removed. As is, we have to manually look at each argument to tell if it was just an added type annotation vs an actual change to the argument. No biggie :)

Reply:
will try to be more restrained in future